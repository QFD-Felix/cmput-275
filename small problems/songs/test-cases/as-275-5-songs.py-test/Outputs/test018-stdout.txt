32 15 13 6 22 14 5 1 25 8 30 28 1 5 37 27 6 6 17 19 5 0 5 1 4 12 4 2 6 37 0 0 16 34 24 6 0 28 19 5 3 12 4 7 1 7 0 0 8 0 14 4 7 1 0 6 6 6 0 1 0 0 0 0 0 0 0 0 0
