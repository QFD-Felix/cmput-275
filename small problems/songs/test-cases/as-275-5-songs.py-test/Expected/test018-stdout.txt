33 16 14 7 23 15 1 5 1 25 8 30 28 1 5 39 27 6 6 17 19 5 0 5 1 4 12 4 2 6 39 0 0 18 36 14 6 0 30 21 5 3 14 4 9 1 9 0 0 1 11 8 0 14 4 7 1 0 6 6 6 0 1 0 0 0 0 0 0 0 0 0
